local/bdrs/application.properties

```diff
@@ -5,5 +5,5 @@ web.http.directory.port=8582
 web.http.directory.path=/api/directory
 # looking up DIDs should not use https
 edc.iam.did.web.use.https=false
-edc.iam.trusted-issuer.issuer.id=did:web:mock-util-service/trusted-issuer
+#edc.iam.trusted-issuer.issuer.id=did:web:mock-util-service/trusted-issuer
```

Comment:
We need to document this somewhere. Else future us (or others) might think: hey let's move the things again to the `application.properties`. They see it fails but will need to do an analysis.

How about removing the outcommented lines in the `.properties`  and add a comment in the compose files stating something like:

"Within docker and docker compose dashs sometimes may not be used in environment-variables (see e.g. https://github.com/docker/compose/issues/12123#issuecomment-2477247374). To be independent of their future change, one can put these variables into the compose files environment variable section)."

Still not ideal but putting it everywhere would be too much.